scripts/prairie_forest.json

```diff
@@ -0,0 +1,1437 @@
+{
+    "citation": "PI: Bryan L. Foster, University of Kansas, bfoster@ku.edu",
+    "description": "Long-term studies of secondary succession and community assembly in the prairie-forest ecotone of eastern Kansas (NSF LTREB # 0950100)",
+    "encoding": "UTF-8",
+    "homepage": "https://foster.ku.edu/ltreb-datasets",
+    "keywords": [
+        "plant biomass exp1",
+        "plant species composition exp1",
+        "plant biomass exp2",
+        "plant species composition exp2"
+    ],
+    "name": "prairie-forest",
+    "resources": [
+        {
+            "dialect": {
+                "delimiter": ",",
+                "header_rows": "1"
+            },
+            "name": "plant_biomass_exp1",
+            "schema": {
+                "fields": [
+                    {
+                        "name": "UID",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Year",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Month",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Unit",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Block",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Gradient",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Plot",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Seed",
+                        "type": "int"
+                    },
+                    {
+                        "name": "N",
+                        "type": "int"
+                    },
+                    {
+                        "name": "P",
+                        "type": "int"
+                    },
+                    {
+                        "name": "Live_Biomass",
+                        "type": "float"
```

Comment:
we don't support float, the script type we use should be double

Reply:
I completed changes that you suggested. The download function works well while the install function still has the previous problem. I checked elements of table again but did not find any problem. I feel it difficult to find the bug and correct it. @zhangcandrew